### DISCUSSION TITLE
is there any way to stop auto assigntment of agent after business hours

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#4860] [Self hosted]
Out business hours suppose a end-user comes to the live chat in website and wants to talk with the agent then is there any way to stop that conversation assigntment to the agent even if there is online agents available ?

like it should check business hours before assign to any agent

![Screenshot (154)](https://user-images.githubusercontent.com/46310938/173595191-0c2eff54-2b9a-48f5-8601-1c6b66d77ab8.png)

**Comment by sojan-official** [comment]
At the moment we don't support this.
